account_invoice_import_invoice2data/wizard/account_invoice_import.py

```diff
@@ -55,9 +60,30 @@ def invoice2data_parse_invoice(self, file_data):
         try:
             invoice2data_res = extract_data(file_name, templates=templates)
         except Exception as e:
-            raise UserError(_("PDF Invoice parsing failed. Error message: %s") % e)
+            raise UserError(_("Invoice parsing failed. Error message: %s") % e)
         if not invoice2data_res:
-            return False
+            if not shutil.which("tesseract"):
+                logger.warning(
+                    "Fallback on tesseract impossible, Could not find the utility. "
+                    "Hint: sudo apt install tesseract-ocr"
+                )
+                return False
+            # Fallback on tesseract
+            logger.info("PDF Invoice parsing failed: Falling back on Tesseract ocr")
```

Comment:
If you channed the log message for Invoice parsing failed on top, can you keep the same logic?

Reply:
@etobella Thanks for the review. 👍 
Sorry, But I don't understand the change you suggesting

Reply:
Reverted the change, as hard coding the file types to PDF.
 
Invoice2data is capable of handling different file formats.
Sadly, we cannot support those different file types.

Because the uploaded file is converted to a base64 string.
When it is passing trough all the account_invoice_import functions, some data got lost.
(file name, mimetype)

Ideally we would have that data available to use in other functions.
(it could improve tesseract ocr results)
But currently we don't have it.

So, I'm reverting the log message.